src/app/ir/conv.ts

```diff
@@ -0,0 +1,73 @@
+import { InOutNode, ModelNode } from '@/app/ir/mainNodes';
+import { Option, UUID } from '@/app/util';
+import {
+  ActivationF, Initializer, Padding, Regularizer,
+} from '@/app/ir/irCommon';
+
+abstract class Conv implements ModelNode, InOutNode {
```

Comment:
I'm not entirely familiar with all the IR but weights & baises are common to a lot of other layers than conv, maybe they should be in another super class like `TrainableLayerNode` or something?

Reply:
Can refactor when it becomes an issue, but overenginneering a hierarchy is very bad practice - I was even considering removing `Conv`